src/MergeContext.js

```diff
@@ -494,7 +494,7 @@ class MergeContext {
     _prHeadSha() { return this._pr.head.sha; }
 
     _prMessage() {
-        return this._pr.title + ' (#' + this._pr.number + ')' + '\n\n' + this._prBody();
+        return (this._pr.title + ' (#' + this._pr.number + ')' + '\n\n' + this._prBody()).trim();
```

Comment:
I think the intent here is not clear. I would replace .trim() with message updates:

```
let message = ...
if (this._prBody().length)
    message += "\n\n" + this._prBody();
return message;
```

but I do not insist on this or a similar change.

Reply:
Left as is because the suggested approach would not work when this._prBody() contains only whitespaces (its length > 0).

Reply:
_prBody() should return trimmed bodies, of course.